.gitignore

```diff
@@ -1,9 +1,138 @@
-.python-version
```

Comment:
I merged #279, so this needs to come back out

Reply:
It's already there in the 'pyenv' section:
https://github.com/vit-project/vit/blob/272126da4e94a64946765df8173cd801fbee68e7/.gitignore#L88

Reply:
Perhaps we're talking past each other. What I meant was that I already merged #279, which includes all these same changes in .gitignore. I think the correct approach given that is:
1. Remove the changes to .gitignore in this PR
2. Rebase this branch against 2.x
3. Force push this branch

Reply:
Sorry for the misunderstanding. The #279 is now excluded from this PR.